packages/frontend/src/components/NavBar/HelpMenu/index.tsx

```diff
@@ -14,15 +14,37 @@ import {
     ItemDescription,
     LinkWrapper,
     MenuWrapper,
+    NotificationWidget,
+    NotificationWrapper,
 } from './HelpMenu.styles';
 
 const HelpMenu: FC = () => {
+    useEffect(() => {
+        //@ts-ignore
+        if (window.Headway) return; // don't load it twice
+        const script = document.createElement('script');
```

Comment:
https://docs.headwayapp.co/widget

Reply:
@rephus we have some window-level scripts initialised here: https://github.com/lightdash/lightdash/blob/1d2678418ba6c0edc1de11dfa6372a9b7aacedbe/packages/frontend/src/providers/AppProvider.tsx#L107-L134 - could you add this with those?

Also it looks like the docs say we should run `init` on every component render: https://docs.headwayapp.co/widget

> In Angular/React you should initialize the widget every time the badge element container is re-rendered.